apps/src/templates/lessonOverview/LessonNavigationDropdown.jsx

```diff
@@ -46,6 +49,20 @@ export default class LessonNavigationDropdown extends Component {
 
   handleDropdownClick = listItem => {
     if (listItem.link) {
+      firehoseClient.putRecord(
+        {
+          study: 'lesson-plan',
+          study_group: this.props.isStudentLessonPlan
+            ? 'student-lesson-plan'
+            : 'teacher-lesson-plan',
+          event: 'navigate-between-lessons',
+          data_json: JSON.stringify({
+            startingLessonId: this.props.lesson.id,
+            endingLessonId: listItem.id
+          })
+        },
+        {includeUserId: true}
+      );
```

Comment:
@tess323 how do you plan to slice this data and/or what is the purpose of collecting this data? the fields within the data_json here will be slower to query.

Reply:
Goal here is to be looking into how folks are using the dropdown.  This is one of the biggest changes from the CB UI so just want to get a general idea of usage.

Reply:
ok. @dmcavoy I would suggest adding `data_int: this.props.lesson.id`. it's a bit speculative, but it might help compute how often this menu gets clicked after a given lesson plan gets loaded (a different metric in the spec).